src/components/totals/totals.html

```diff
@@ -3,5 +3,7 @@
   <span class="report-header-text scorecard-item">D/F: {{faultMarks$ | async}}</span>
   <span class="report-header-text scorecard-item">S: {{seriousMarks$ | async}}</span>
   <span class="report-header-text scorecard-item">D: {{dangerousMarks$ | async}}</span>
-  <ion-icon (click)="undoMark()" name="undo"></ion-icon>
+  <button class="report-header-text scorecard-item">
```

Comment:
Can we make this button different from the other scorecard items? Perhaps a filled button like 'End'? What do you guys think?

Reply:
hym, it would be to busy if we do another big button next to End button?

Reply:
I was just thinking colour, size can stay the same.